src/main/java/org/ow2/proactive/catalog/callgraph/CatalogObjectCallGraphPDFGenerator.java

```diff
@@ -79,8 +81,6 @@
 
     private static final String MAIN_TITLE = "ProActive Call Graph Report";
 
-    private CallGraphHolder callGraphHolder = new CallGraphHolder();
```

Comment:
You should update in accordance the tests of this class so you can include CallGraphHolder class removal in part of this pull request.

Reply:
Ok, I hadn't seen the new occurrence to that class introduced by your commit. I validate your PR.